packages/charts/src/components/chart-tooltip/chart-tooltip.utils.ts

```diff
@@ -1,4 +1,6 @@
 import { Margins, Dimension } from '../../types';
+import { ARROW_SIZE } from '@keen.io/ui-core';
```

Comment:
In all files we are trying to keep `imports` convention:
- External npm modules
- Keen npm modules
- Local imports

Could You please adjust the order of that imports to this convention.